test262/main.js

```diff
@@ -304,6 +371,18 @@ test262@${test262Version}, test262-parser-tests@${test262ParserTestsVersion}`;
       document.getElementById("chart-test262-parser-tests"),
       charts["test262-parser-tests"]
     );
+    initializeChart(
+      document.getElementById("chart-test262-performance-tests"),
+      charts["test262-performance-tests"],
+      "Time",
+      TestResultLabels[TestResult.DURATION]
```

Comment:
using the above suggestion we can then do:

```suggestion
      { xAxisTitle: "Time", yAxisTitle: TestResultLabels[TestResult.DURATION] },
```